src/styles/Colors.scss

```diff
@@ -8,3 +8,4 @@ $gray: gray;
 $mainTheme: linear-gradient(0deg, rgba(40, 103, 178, 0.28895308123249297) 0%, rgba(255, 255, 255, 1) 71%);
 $white: white;
 $black: black;
+$linkedinMainTheme: #F3F2EF;
```

Comment:
Please use consistent MainTheme name (there is should be not too much main themes)